mdpow/equil.py

```diff
@@ -206,10 +211,23 @@ def __init__(self, molecule=None, **kwargs):
                                          box=self.solventmodel.coordinates,
                                          distance=distance)
 
-            self.filename = filename or self.solvent_type+'.simulation'
+            self.filename = filename or self.solvent_type + '.simulation'
 
         super(Simulation, self).__init__(**kwargs)
 
+    @staticmethod
+    def load_pickle():
```

Comment:
For compatibility between 2 and 3, could probably also use six

Reply:
six is preferred — better use existing, tested libraries and six is standard and already used